packages/prompts/src/index.ts

```diff
@@ -759,3 +759,33 @@ export const group = async <T>(
 
 	return results;
 };
+
+export type Task = {
+	/**
+	 * Task title
+	 */
+	title: string;
+	/**
+	 * Task function
+	 */
+	task: (message: (string: string) => void) => string | Promise<string> | void | Promise<void>;
+
+	/**
+	 * If enabled === false the task will be skipped
+	 */
+	enabled?: boolean;
+};
+
+/**
+ * Define a group of tasks to be executed
+ */
+export const tasks = async (tasks: Task[]) => {
+	for (const task of tasks) {
+		if (task.enabled !== false) {
+			const s = spinner();
+			s.start(task.title);
+			const result = await task.task(s.message);
+			s.stop(result || task.title);
+		}
```

Comment:
```suggestion
		if (!task.enabled) continue;
		
		const s = spinner();
		s.start(task.title);
		const result = await task.task(s.message);
		s.stop(result || task.title);
```

A small suggestion as it's easier to read but Good work  @artegoser 🙏🏼

Reply:
Hello, but wouldn't you have to specify enabled property within each task? Maybe it would be better to do 
```ts
if (task.enabled === false) continue;
```

In this case, the task will be skipped only if the user explicitly specified false.